packages/theme/package.json

```diff
@@ -48,13 +48,10 @@
     "win32"
   ],
   "oclif": {
-    "commands": "dist/cli/commands",
-    "hooks": {
-      "init": "@shopify/cli-kit/node/hooks/init"
-    }
```

Comment:
Is removing this intended?

Reply:
I don't think this is right. It needs to be run twice because there are 2 copies of cli-kit, one for `@shopify/cli-main` and one for `@shopify/theme`. This is an artifact of the CLI being installed globally.